Scripts/UpdateRemoteDesktopSettings/Set-DTL_RemoteDesktopSettings_ps6.ps1

```diff
@@ -0,0 +1,90 @@
+﻿<#
```

Comment:
Same comments apply to this script file as well. Consider finding a way to combine these 2 scripts into a single one. It makes it a lot easier to maintain.

If you need to maintain conditional logic for PS5 vs PS6, consider checking `$PSVersionTable.PSVersion.Major`.

Reply:
Ok, thanks!  The issue isn't the version of Powershell, but rather the AzureRM Powershell commandlets.  good advice though, let's combine it!  The command to get the major version of Azure powershell is:
(Get-Module -ListAvailable -Name Azure -Refresh).Version.Major

I'm updating this now.